gobrew.go

```diff
@@ -499,9 +504,18 @@ func (gb *GoBrew) Upgrade(currentVersion string) {
 	}(source)
 
 	goBrewFile := filepath.Join(gb.installDir, "bin", "gobrew"+fileExt)
-	if err = os.Remove(goBrewFile); err != nil {
-		utils.Errorf("[Error] Cannot remove binary file: %s", err.Error())
-		return
+	if runtime.GOOS == "windows" {
+		goBrewOldFile := goBrewFile + ".old"
+		_ = os.Remove(goBrewOldFile)
```

Comment:
You are deleting a file that may not be there. Why don't you check for its presence in the directory?
It's not very good to just ignore the error if it occurs.

Reply:
I forgot to remove that call, it is not necessary on Windows because the rename uses the option MOVEFILE_REPLACE_EXISTING when renaming the file, so it will overwrite the old file if necessary.